### PR TITLE
adds information about the cron log

### PR DESCRIPTION
**Pull request by otaviojava**
It is important if you add the information where the cron log is somehow.

### CODE REVIEW

File: docs/src/configuration/app/cron.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ Cron runs are executed using the dash shell, not the bash shell used by normal S
 
 If an application defines both a `web` instance and a `worker` instance, cron tasks will be run only on the `web` instance.
 
+> Different from the application, the Cron job log is on the `/var/log/cron.log`.
```
Comment (chadwcarlson) [inline docs/src/configuration/app/cron.md:22]:
```suggestion
{{< note >}}
Cron logs can be found within the application container at `/var/log/cron.log`.
{{< /note >}}
```

Comment (chadwcarlson) [reply docs/src/configuration/app/cron.md:22]: Slight re-wording, and uses the note shortcode

Comment (Crell) [inline docs/src/configuration/app/cron.md:22]:
```suggestion
Cron log output is captured in the `/var/log/cron.log` file.  See the [Log page]({{< relref "/development/logs/md" >}}) for more information on logging.
```

Comment (otaviojava) [reply docs/src/configuration/app/cron.md]: Done

File: docs/src/configuration/app/cron.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ Cron runs are executed using the dash shell, not the bash shell used by normal S
 
 If an application defines both a `web` instance and a `worker` instance, cron tasks will be run only on the `web` instance.
 
+{{< note >}}
+Cron log output is captured in the at `/var/log/cron.log`.  See the [Log page]({{< relref "/development/logs/md" >}}) for more information on logging.
```
Comment (Crell) [inline docs/src/configuration/app/cron.md:23]:
```suggestion
Cron log output is captured in the at `/var/log/cron.log`.  See the [Log page]({{< relref "/development/logs.md" >}}) for more information on logging.
```

**Pull request merged**